ets-poms-application-parent/pom.xml

```diff
@@ -10,18 +10,71 @@
     <packaging>pom</packaging>
     <name>EShop Technology Stack (ETS) - Application Parent POM</name>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>net.alchim31.maven</groupId>
-                <artifactId>scala-maven-plugin</artifactId>
-            </plugin>
-            <plugin>
-                <groupId>org.scalatest</groupId>
-                <artifactId>scalatest-maven-plugin</artifactId>
-            </plugin>
-        </plugins>
-    </build>
+    <profiles>
+        <profile>
+            <id>ets-application-main</id>
+            <activation>
+                <file>
+                    <exists>src/main</exists>
+                </file>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>net.alchim31.maven</groupId>
+                        <artifactId>scala-maven-plugin</artifactId>
+                    </plugin>
+                    <plugin>
```

Comment:
Can be removed because auto activated by ets-application-test (scala test runner will not run tests in src/main/scala)